src/main/java/step3/Car.java

```diff
@@ -6,15 +6,34 @@
     private static final int MOVE_CRITERION = 4;
 
     private int moveDistance;
+    private String carName;
 
-    public Car() {
+    public Car(String carName) {
         this.moveDistance = 0;
+        validateCarName(carName);
+        this.carName = carName;
+    }
+
+    public Car(String carName, int moveDistance) {
+        this.moveDistance = moveDistance;
+        validateCarName(carName);
+        this.carName = carName;
+    }
```

Comment:
다음과 같이 주 생성자를 사용해보면 어떨까요? 😄 
```suggestion
    public Car(String carName, int moveDistance) {
        this.moveDistance = moveDistance;
        validateCarName(carName);
        this.carName = carName;
    }

    public Car(String carName) {
        this(carName, 0);
    }
```